### PR TITLE
Updating gitignore for more robust coverage

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updating gitignore for more robust coverage
 
## Additions
 
- Added new Compiled Source group of filetypes
- Added new Package group of filetypes
- Added new Logs and databases group of filetypes
 
## Removals
 
None
 
## Changes
 
- Added `.env` files to Python group
- Added `src/vendor/` dir to Front End group
- Added `/dist` dir to Front End group
 
## Testing
 
None

## Review
 
- @Scotchester 
- @ascott1 
- @contolini 
- @marcesher 
et al
 
## Preview

None
 
## Notes
 
- All of these filetypes come from the existing cfgov-refresh project and experience updating that project with the CF generator

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +68,5 @@ coverage.xml
 node_modules/
 bower_components/
 .grunt/
+src/vendor/
+/dist
```
Comment (Scotchester) [inline .gitignore]: Should probably be `dist/` to match the format of the others.

**Comment by Scotchester** [comment]
Other than the one comment, looks good to me. I'd like to have at least one other person weigh in before merging, though.

**Comment by ascott1** [comment]
@Scotchester's comment makes sense to me. Otherwise, this looks great. +1 to merge

File: .gitignore (hunk 1 above)
Comment (jimmynotjim) [inline .gitignore]: Good catch, fixed and updated

**Comment by Scotchester** [comment]
Thanks!

**Pull request merged**